src/CodebaseTree/NamespaceListing.elm

```diff
@@ -149,15 +149,15 @@ decodeContent parentFqn =
                 (Decode.map3 TypeListing
                     (field "typeHash" Hash.decode)
                     (field "typeName" (FQN.decodeFromParent parentFqn))
-                    (Decode.map Category.Type (Type.decodeTypeCategory "typeTag"))
+                    (Decode.map Category.Type (Type.decodeTypeCategory [ "typeTag" ]))
                 )
 
         decodeTermListing =
             Decode.map SubDefinition
                 (Decode.map3 TermListing
                     (field "termHash" Hash.decode)
                     (field "termName" (FQN.decodeFromParent parentFqn))
-                    (Decode.map Category.Term (Term.decodeTermCategory "termTag"))
+                    (Decode.map Category.Term (Term.decodeTermCategory [ "termTag" ]))
```

Comment:
Some general improvements were made to decoders — most of them now take a path instead of a field name.